src/PrettierEditProvider.ts

```diff
@@ -1,26 +1,33 @@
 import {
     workspace,
+    window,
     DocumentRangeFormattingEditProvider,
     DocumentFormattingEditProvider,
     Range,
     TextDocument,
     FormattingOptions,
     CancellationToken,
-    TextEdit
+    TextEdit,
+    Selection,
+    Position
 } from 'vscode';
 
 const prettier = require('prettier');
 
+type ShowAction = "Show";

```

Comment:
Was wondering if _Show_ was a good text for the action. I also had _Go to_ in mind.
Any preference ?